uber/site_sections/preregistration.py

```diff
@@ -254,9 +244,6 @@ def prereg_payment(self, session, payment_id, stripeToken):
                 if attendee.amount_extra:
                     attendee.amount_paid = attendee.total_cost
             session.add(group)
```

Comment:
This passes unit tests, so hopefully that means the change didn't break anything. :'D